sda-auth/main.go

```diff
@@ -389,6 +389,7 @@ func main() {
 	app.Get("/elixir/s3conf", authHandler.getElixirConf)
 	app.Get("/elixir/login", authHandler.getElixirLogin)
 	app.Get("/elixir/cors_login", authHandler.getElixirCORSLogin)
+	app.Get("/info", authHandler.getInfo)
```

Comment:
This endpoint should only be active if the service has been properly configured, i.e it has a crypt4gh public key.